docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+# Docker Compose file for FreeCodeCamp
+#
+# Bootstrap with:
+#   docker-compose run --rm server npm install
+#   docker-compose run --rm server npm run only-once
+#
+# Run with:
+#   docker-compose up
+#
+version: "3"
+services:
+    db:
+        image: mongo:3.2.6
+        ports:
+            - "27017:27017" # Change port if needed
+    freecodecamp:
+        image: node:8.9.4
+        depends_on:
+            - db
+            - mailhog
+        environment:
+            - MONGOHQ_URL=mongodb://db:27017/freecodecamp
+        volumes:
+            - .:/app
```

Comment:
As was changed in the most recent PR. It's a little nicer than fcc, so kept that in.